waku/v2/node/storage/peer/waku_peer_storage.nim

```diff
@@ -82,19 +84,20 @@ proc new*(T: type WakuPeerStorage, db: SqliteDatabase): PeerStorageResult[T] =
 method put*(db: WakuPeerStorage,
             peerId: PeerID,
             storedInfo: StoredInfo,
-            connectedness: Connectedness): PeerStorageResult[void] =
+            connectedness: Connectedness,
+            disconnectTime: int64): PeerStorageResult[void] =
 
   ## Adds a peer to storage or replaces existing entry if it already exists
   let prepare = db.database.prepareStmt(
-    "REPLACE INTO Peers (peerId, storedInfo, connectedness) VALUES (?, ?, ?);",
-    (seq[byte], seq[byte], int32),
+    "REPLACE INTO Peer (peerId, storedInfo, connectedness, disconnectTime) VALUES (?, ?, ?, ?);",
```

Comment:
This is outside of scope of PR, but I wonder if we want to do REPLACE here? Might make sense to make this an append-log and then take last entry. The historical state could be useful to understand connect history, perhaps?

Reply:
True. And then rather perform `UPDATE` during a single connection session, which will likely be more efficient.